modules/dxp/apps/portal-workflow/portal-workflow-metrics-web/src/main/resources/META-INF/resources/js/components/instance-list-page/InstanceListPageHeader.es.js

```diff
@@ -47,6 +53,16 @@ export default function Header({
 	const {openModal} = useContext(ModalContext);
 	const previousCount = usePrevious(totalCount);
 
+	const previousFetchData = usePrevious(fetchData);
```

Comment:
@ramonmsp could you add a line between the definition and the usage of previousFetchData? Thanks :)